src/reducers/contacts.reducer.js

```diff
@@ -10,6 +11,8 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case FETCH_CONTACTS:
       return { ...state, contacts: randomContactsList };
+    case FETCH_FAVOURITES:
+      return { ...state, favourites: [...getFavouritesList(state.contacts)] };
```

Comment:
I don't think you need the spread operator here because you are already doing that inside the getFavouritesList in the first row.
So i guess you can just do this:
`return { ...state, favourites: getFavouritesList(state.contacts) };`

This is my only comment, so once you fix this, you can merge it.